server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/model/message/view/MessageFullViewFactoryTest.java

```diff
@@ -250,7 +251,7 @@ void headersShouldBeSetIntoMessage() throws Exception {
                 .subject("test subject")
                 .date(Instant.parse("2015-07-14T12:30:42.000Z"))
                 .size(headers.length())
-                .preview("(Empty)")
+                .preview(Preview.NO_BODY)
```

Comment:
please don't reference prod constant in tests if it's something we want to assert